app_dart/lib/src/service/gerrit_service.dart

```diff
@@ -65,12 +69,30 @@ class GerritService {
         Uri.https('${slug.owner}.googlesource.com', '${slug.name}/+log/refs/heads/$branch', <String, String>{
       'format': 'json',
     });
-    final Map<String, dynamic> response = await _get(url) as Map<String, dynamic>;
+    final Map<String, dynamic> response = await _getJson(url) as Map<String, dynamic>;
     final List<dynamic> commitsJson = response['log'] as List<dynamic>;
 
     return commitsJson.map((dynamic part) => GerritCommit.fromJson(part as Map<String, dynamic>));
   }
 
+  /// Gets the info for a commit from a remote git repository using the gerrit APIs.
+  ///
+  /// See more:
+  ///   * https://gerrit-review.googlesource.com/Documentation/rest-api-projects.html#get-commit
+  Future<GerritCommit?> getCommit(RepositorySlug slug, String projectName, String commitId) async {
```

Comment:
projectName is a part of the `RepositorySlug`. Should you update the caller to use `RepositorySlug('flutter', 'mirrors/$project')`?

Reply:
Yup, good catch.